istioctl/cmd/root.go

```diff
@@ -75,8 +76,12 @@ var (
 	// Create a kubernetes client (or mockClient) for talking to control plane components
 	kubeClientWithRevision = newKubeClientWithRevision
 
+	// The kubeClient used throughout the cmd package. After kubeClient is initialized for the first
+	// time the client is not initialized again and is shared between callers.
+	kubeClient kube.CLIClient
+
 	// Create a kubernetes.ExecClient (or mock) for talking to data plane components
-	kubeClient = newKubeClient
+	initKubeClient = newKubeClient
```

Comment:
nit: naming-wise, `getKubeClient` makes more sense; you're not initializing the client each time. You're effectively memoizing the value here. Also, why not just rename `newKubeClient` to `initKubeClient`?

Reply:
> you're not initializing the client each time

You have a good point. I wanted to highlight the fact this was initializing a variable but like you said, that only happens once so it doesn't really make sense here. Will update!

I agree with `newKubeClient` being called `initKubeClient` too. Good call 👍